### ISSUE TITLE
Details about weekend stop are missing

### ISSUE THREAD

**Issue by juanchodepisa** [online-go/online-go.com#142]
### Summary

When one receives a challenge, is playing a game, or wants to join a tournament, it is often useful to know what the full settings are.

I've found that for correspondence games, the details about weekend stop are missing in the following contexts:

- When playing a game and clicking on game details
- When viewing the details of an open challenge
- When receiving a direct challenge
- When viewing the details of an open tournament

Please add that information

**Comment by mlopezviedma** [comment]
When playing a game and clicking on game details it's already there. E.g. https://online-go.com/game/7832568

**Comment by juanchodepisa** [comment]
Well, I beg to disagree. Where does it ever say anything here https://online-go.com/game/7969251 or here https://online-go.com/tournament/25404?

As a matter of fact, I didn't even know ladder games had no weekend stop. Had to dig up that information from old forum posts, but only after I made sure my games were actually set to no-weekend-stop, which wasn't easy.

The only way to know for sure was to make a query through the API: https://online-go.com/api/v1/games/7969251, and then look that "pause_on_weekends" was set to false.

I cannot say the same for tournaments, for even this https://online-go.com/api/v1/tournaments/25404 yields nothing.

In other words, saying when *there is* a weekend stop but not when *there isn't* is weird and confusing. The weekend stop has always been a thing on correspondence servers, so it's an integral part of their time settings (whether it's present or it isn't).

______________

On a related side note, I don't see why it would be relevant for games with no-time-limits to have a pause on weekends (Eg: https://online-go.com/game/8002139)

**Comment by juanchodepisa** [comment]
And now this game https://online-go.com/game/8035489 is pausing on weekend.

Nowhere does it say it should, let alone the fact that it is from a sitewide tournament, which I had understood were all set to not have weekend stops.

**Comment by roy7** [comment]
Interesting. it doesn't say pauses on weekends. Yet this game that does say it:

https://forums.online-go.com/t/is-there-no-weekend-pause-anymore/11571

Didn't pause and the player lost on timeout.

Is the display of "pauses on weekends" perhaps reversed? In fact, yes, looking at live bot games they say "Pauses on weekends" which I know they don't.

**Comment by anoek** [comment]
Following up on the pause on weekends thing for those games, turns out that was a bug in the tournament game creation code where we used to set that flag for any correspondence game. I'm switching it over to use the time control parameters like everything else now, so future games will be created correctly.
